scripts/st2bootstrap-deb.sh

```diff
@@ -14,6 +14,7 @@ ST2WEB_PKG_VERSION=''
 ST2CHATOPS_PKG_VERSION=''
 USERNAME=''
 PASSWORD=''
+SUBTYPE=`lsb_release -a 2>&1 | grep Codename | grep -v "LSB" | awk '{print $2}'`
```

Comment:
lsb_release isn't always installed. Best to quit with an error saying if SUBTYPE not in [xenial, trusty]